### PR TITLE
bookings global export, extended filters

### PR DESCRIPTION
**Pull request by folix-01**

### CODE REVIEW

File: src/redturtle/prenotazioni/browser/bookings_export.py [hunk 1]
```diff
@@ -145,19 +186,65 @@ def get_csv(self):
...
+
+        if booking_creation_from:
+            try:
+                self.booking_creation_from = datetime.datetime.fromisoformat(
+                    booking_creation_from
                 )
             except ValueError:
-                raise BadRequest(_("Bad date format passed"))
+                raise BadRequest(
+                    api.portal_translate(
```
Comment (cekk) [inline src/redturtle/prenotazioni/browser/bookings_export.py:231]: I don't see *.po updated

File: src/redturtle/prenotazioni/browser/bookings_export.py [hunk 2]
```diff
@@ -49,25 +53,62 @@ def csv_fields(self):
...
-            portal_type="Prenotazione",
-            Date={
-                "query": (
-                    get_default_timezone(True).localize(self.date),
-                    get_default_timezone(True).localize(
-                        self.date.replace(hour=23, minute=59)
+        if created:
+            return api.portal.get_tool("portal_catalog").unrestrictedSearchResults(
+                portal_type="Prenotazione",
+                Date={
```
Comment (cekk) [inline src/redturtle/prenotazioni/browser/bookings_export.py:86]: Why search on Date index? 
Date is effective date or modified date (if item is not published yet).
Shouldn't be better searching for booking date? Or Date index is filled with booking date for bookings?

**Review by cekk — CHANGES_REQUESTED** [review]

File: src/redturtle/prenotazioni/browser/bookings_export.py (hunk 2 above)
Comment (folix-01) [reply src/redturtle/prenotazioni/browser/bookings_export.py:86]: is filled with booking date

File: src/redturtle/prenotazioni/browser/bookings_export.py (hunk 1 above)
Comment (folix-01) [reply src/redturtle/prenotazioni/browser/bookings_export.py:231]: Done

File: src/redturtle/prenotazioni/locales/it/LC_MESSAGES/redturtle.prenotazioni.po [hunk 3]
```diff
@@ -47,9 +51,21 @@ msgstr "L'orario di inizio del pomeriggio non può essere successivo alla chiusu
 msgid "Attention"
 msgstr "Attenzione!"
 
-#: redturtle/prenotazioni/browser/bookings_export.py:160
-msgid "Bad date format passed"
-msgstr "Parametro 'date' è composto male"
+#: redturtle/prenotazioni/browser/bookings_export.py:232
+msgid "Badly composed `booking_creation_from` value"
+msgstr "Valore del campo `booking_creation_from` è composto male."
```
Comment (cekk) [inline src/redturtle/prenotazioni/locales/it/LC_MESSAGES/redturtle.prenotazioni.po:56]:
```suggestion
msgstr "Il valore del campo `booking_creation_from` non è corretto."
```

e così anche gli altri

**Review by cekk — CHANGES_REQUESTED** [review]

File: src/redturtle/prenotazioni/locales/it/LC_MESSAGES/redturtle.prenotazioni.po (hunk 3 above)
Comment (folix-01) [reply src/redturtle/prenotazioni/locales/it/LC_MESSAGES/redturtle.prenotazioni.po:56]: grandeeee

fatto

**Review by cekk — APPROVED** [review]

**Pull request merged**